[{"ContainerToolsException.Message":"Échec de la commande Docker. Code de sortie : 1. Get \"https://mcr.microsoft.com/v2/dotnet/aspnet/manifests/sha256:8fb8872ca3c12f6cfa27f6d0c8d5581f2b813ef44d7cf093c25c23b93769f459\": net/http: TLS handshake timeout","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   à Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__5.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__11.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__184.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__26.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"Dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"Get \"https://mcr.microsoft.com/v2/dotnet/aspnet/manifests/sha256:8fb8872ca3c12f6cfa27f6d0c8d5581f2b813ef44d7cf093c25c23b93769f459\": net/http: TLS handshake timeout"}]